fab/fabfile.py

```diff
@@ -728,6 +730,18 @@ def force_update_static():
     execute(services_restart)
 
 
+def _tag_commit():
+    sh.git.fetch("origin", env.code_branch)
+    deploy_time = datetime.datetime.utcnow()
+    tag_name = "{:%Y-%m-%d_%H.%M}-{}-deploy".format(deploy_time, env.environment)
```

Comment:
This makes tags which look like:
`2015-07-01_19.30-staging-deploy`
git won't let you use spaces or ":" in tag names, so that timestamp is kinda ugly